src/main/java/org/drtshock/Potato.java

```diff
@@ -87,6 +88,9 @@ public boolean isPutIntoOven() throws OvenException {
         }
     }
 
+    public void isAncientPotato() {
```

Comment:
This should return a `boolean`. If you want there to be a message, consider throwing an exception.